addons/api/addon/serializers/managed-group.js

```diff
@@ -11,6 +11,10 @@ import {
 
 export default class ManagedGroupSerializer extends ApplicationSerializer {
   // =properties
+  /**
+   * @type {boolean}
+   */
+  serializeScopeID = false;
```

Comment:
the scope_id should not be sent as part of the request body for managed-groups

Reply:
ty for adding this back!